### PR TITLE
N°2039 - Update Chinese translations for the newsroom

### PR DESCRIPTION
**Pull request by chileeb**
As following picture shows:
image
image

### CODE REVIEW

**Comment by chileeb** [comment]
@annProg @bdejin Please Review.

**Comment by annProg** [comment]
对比 ActionEmail 翻译成 邮件通知，Action应该不需要直译。ActioniTopNotification 字面意思大概是iTop通知，可能类似于站内信概念

**Comment by chileeb** [comment]
> 对比 ActionEmail 翻译成 邮件通知，Action应该不需要直译。ActioniTopNotification 字面意思大概是iTop通知，可能类似于站内信概念

好主意。

File: dictionaries/en.dictionary.itop.core.php [hunk 1]
```diff
@@ -629,8 +629,12 @@
 //
 
 Dict::Add('EN US', 'English', 'English', array(
+	'ActioniTopNotification:content' => 'ActioniTopNotification:content',
+	'ActioniTopNotification:settings' => 'ActioniTopNotification:settings',
```
Comment (Molkobain) [inline dictionaries/en.dictionary.itop.core.php]: @steffunky these were missing from the original commit or are they elsewhere?

File: dictionaries/zh_cn.dictionary.itop.core.php [hunk 2]
```diff
@@ -478,6 +478,41 @@
...
+	'Class:EventiTopNotification' => ITOP_APPLICATION_SHORT.' 通知',
+	'Class:EventiTopNotification+' => '',
+	'Class:EventiTopNotification/Attribute:title' => '标题',
+	'Class:EventiTopNotification/Attribute:title+' => '',
+	'Class:EventiTopNotification/Attribute:icon' => 'Icon',
+	'Class:EventiTopNotification/Attribute:icon+' => '',
+	'Class:EventiTopNotification/Attribute:priority' => '关键',
+	'Class:EventiTopNotification/Attribute:priority+' => '关键',
+	'Class:EventiTopNotification/Attribute:priority/Value:1' => '紧急',
+	'Class:EventiTopNotification/Attribute:priority/Value:1+' => 'Critical',
```
Comment (Molkobain) [inline dictionaries/zh_cn.dictionary.itop.core.php]: This is not translated, is it on purpose?

File: dictionaries/zh_cn.dictionary.itop.core.php [hunk 3]
```diff
@@ -587,6 +622,39 @@
...
 ));
 
+//
+// Class: ActioniTopNotification
+//
+
+Dict::Add('ZH CN', 'Chinese', '简体中文', array(
+	'ActioniTopNotification:content' => '内容',
+	'ActioniTopNotification:settings' => '设置',
+	'Class:ActioniTopNotification' => 'iTop通知',
```
Comment (Molkobain) [inline dictionaries/zh_cn.dictionary.itop.core.php]:
```suggestion
	'Class:ActioniTopNotification' => ITOP_APPLICATION_SHORT.'通知',
```

**Review by Molkobain — CHANGES_REQUESTED** [review]

File: dictionaries/en.dictionary.itop.core.php (hunk 1 above)
Comment (Molkobain) [inline dictionaries/en.dictionary.itop.core.php]:
```suggestion
	'ActioniTopNotification:content' => 'Content',
	'ActioniTopNotification:settings' => 'Settings',
```

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
All good for me! Is it all good for you as well @chileeb @annProg @bdejin ?

**Comment by chileeb** [comment]
> All good for me! Is it all good for you as well @chileeb @annProg @bdejin ?

That's good!

**Comment by Molkobain** [comment]
Accepted during functional review. Just keep in mind that the newsroom feature is not finished and dict entries may change.

**Pull request merged**